### ISSUE TITLE
'module' object has no attribute 'QtOpenGL'

### ISSUE THREAD

**Issue by jonntd** [mottosso/Qt.py#176]
maya 2017 

from Qt import *
# Error: AttributeError: file <maya console> line 1: 'module' object has no attribute 'QtOpenGL' #

**Comment by mottosso** [comment]
Thanks for reporting this @jonntd.

This stems from an issue (bug) with the version of PySide2 in 2017, it has `QtOpenGL` in it's [`__all__` variable](https://github.com/pyside/pyside2/blob/master/PySide2/__init__.py.in#L1), but no access to it physically.

That is, you get the same result with PySide2.

```python
from PySide2 import *
# Error: AttributeError: file <maya console> line 1: 'module' object has no attribute 'QtOpenGL' # 
```

That said, there is a solution and fix for this in a recent PR here: #173 

Once this is merged, this problem should be no more.

**Comment by mottosso** [comment]
As a workaround, you can call this.

```python
from Qt import QtWidgets, QtCore, QtGui
```

**Comment by mottosso** [comment]
Fixed in 1.0.0.b1
